extensions/amp-accordion/1.0/amp-accordion.js

```diff
@@ -128,6 +129,7 @@ const bindSectionShimToElement = (element) => SectionShim.bind(null, element);
  */
 function HeaderShim(sectionElement, {onClick}) {
   const headerElement = sectionElement.firstElementChild;
+  headerElement.classList.add('i-amphtml-accordion-header');
```

Comment:
This is a side-effect. Let's move it to the `useLayoutEffect` below. Alternatively, we can move this code to `getState`.

Reply:
Updated.